src/App.js

```diff
@@ -25,10 +18,27 @@ function App() {
     };
   }, []);
 
+  const mockUser = {
+    username: 'john123',
+    name: 'John Doe',
+    email: 'johndoe@example.com',
+    bio: 'The song that I am loving this week is Munekita.',
+    roles: [
+      { id: 1, name: 'User' },
+      { id: 2, name: 'Admin' },
+    ],
+    image: 'https://cdn.dribbble.com/users/6142/screenshots/5679189/media/1b96ad1f07feee81fa83c877a1e350ce.png?compress=1&resize=400x300&vertical=top',
```

Comment:
Make sure to check with the backend team to get a good idea of what fields we expect to get. This looks mostly okay, but I don't think there's a `bio` field and I don't think you can have more than one `role` so it shouldn't be an array.